CONTRIBUTING.md

```diff
@@ -0,0 +1,112 @@
+# Contributing to the website hosted at Dogecoin.com
+
+Dogecoin.com is a community-driven effort. The purpose of the website is to offer information and education about Dogecoin, advocate for its adoption, and inform users about safe practices.
+
+Dogecoin.com aims to provide a service to Dogecoin users and to be inclusive. As such, submissions to this repository cannot be an expression of the interests of specific organizations, companies, or individuals.
+
+## Contributor Workflow
+
+The repository is maintained using the "contributor workflow" where everyone without exception contributes to the website using "pull requests". This facilitates social contribution, easy testing, and peer review.
+
+As the repository contains both code contributions and content, the workflow accommodates and facilitates both.
+
+To contribute content to the website, the workflow is as follows:
+
+  - Create a branch within which you will work on your changes.
+  - Commit changes to the branch.
+  - Raise a Pull Request via GitHub.
+
+To contribute significant structural and code changes to the website, the workflow is as follows:
+
+  - Fork the repository in GitHub, and clone it in your development machine.
+  - Create a topic branch, within which you will work on your changes.
+  - Commit changes to the branch.
+  - Raise a Pull Request via GitHub.
```

Comment:
This may be out of scope for this document, but is there a preference between creating branches in this repository versus a fork?

Reading these two sections makes me wonder if there's an assumption about the size or scope of a contribution that changes whether it's desirable to fork the repository. If such an assumption exists, it might be useful to make it clear in this document.

Reply:
So, this is a good point, of course. 

There is a reason why I wrote it like that: this is how many (not all) content writers have contributed so far, and it makes it easier for them to propose edits and new articles directly from Github. which automatically creates a branch for the user. This makes it very easy to edit and propose changes even for users that are new to git and Github.

I have seen that many who want to contribute content and that do not have a developer background tend to postpone contributing because they are afraid of git and Github (Github Desktop helps a bit with that).

I know it may seem like a small thing, and in the end I see we need to go the way you suggest - just pointing out the rationale. I personally think it is a good thing for people to get used to git and learn to use it, but at the same time lowering the barrier of entry can help too.

![Capture](https://user-images.githubusercontent.com/92135099/211222721-fb391059-42d6-4922-a47a-db90cde74a01.PNG)

Reply:
Ah! That makes sense! So I suppose I can just cross it out from the Guidelines.

Thank you!